app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function name `copy_file` is a bit misleading because it gives the impression that it's expecting a file as an argument, but it's actually expecting a command. Consider renaming it to something more descriptive, like `copy_file_from_command`.